vtex-apps/search-result
# Feature: Custom product summary at interval

#### What problem is this solving?

- Adds the ability to render a custom instance (Slot) of a `product-summary` in search result at certain interval. This saves a lot of time writing CSS to achieve similar results and is more flexible allowing to specify every aspect of the custom summary child blocks as well.

#### How to test it?

1. Go to: [Workspace](https://fashioneurope--beightoneagency.myvtex.com/mercearia)
1. Observe that a custom summary is rendered at every 4 items.

#### Screenshots or example usage:

```jsonc
{
  "gallery": {
    "blocks": ["product-summary.shelf#search"],
    "props": {
      "maxItemsPerRow": 4,
      "customSummaryInterval": 5,
      "CustomSummary": "product-summary.shelf#custom"
    }
  }
}
```

![department](https://user-images.githubusercontent.com/1207017/101687291-0cff1780-3a49-11eb-9c00-678b70001c8a.jpg)

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/U18bCh0yoDO2Q/giphy.gif)
